src/App.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <h1>MEMORY GAMME</h1>
+  <button @click="openFullScreen" class="fullScreenButton"></button>
+  click me
```

Comment:
`Click me` nic nie mówi o tym buttonie
```suggestion
Open fullscreen
```